### PR TITLE
Change language(ko) expression && Warning msg(Don't make functions within a loop)

### PR DESCRIPTION
**Pull request by ctyvx784**
I fixed several sentences to naturally in lang/ko/strings.po.

### CODE REVIEW

**Comment by ctyvx784** [comment]
Additionally, I fixed the correct line number that pointing script position in annotation

File: script/outside.js [hunk 1]
```diff
@@ -333,6 +326,16 @@ var Outside = {
 		}
 	},
 	
+	workers_call: function(i){
+		var child = $(this);
+		var cName = child.children('.row_key').text();
+		if(cName != 'gatherer') {
+			if(cName < lk) {
+				curPrev = child.attr('id');
```
Comment (Continuities) [inline script/outside.js]: `curPrev` is undefined here, as it was previously coming from the closure scope of the inline function.

File: script/room.js [hunk 2]
```diff
@@ -886,6 +881,14 @@ var Room = {
 		}
 	},
 	
+	location_call: function(i){
+		var child = $(this);
+		var cName = child.children('.row_key').text();
+		if(cName < lk) {
+			curPrev = child.attr('id');
```
Comment (Continuities) [inline script/room.js]: Same `curPrev` problem

File: lang/ko/strings.po [hunk 3]
```diff
@@ -2956,7 +2956,7 @@ msgstr "이 작은 마을은 불타기 시작한지 얼마 안된 게 분명하
 #: script/events/setpieces.js:2527
 msgid ""
 "the bodies of the wanderers that lived here are still visible in the flames."
-msgstr "여기 살던 떠돌이들이 시체가 아직도 불길 속에서 보인다."
+msgstr "여기 살았던 떠돌이들의 시체가 아직도 불길 속에서 보인다."
```
Comment (youknowone) [inline lang/ko/strings.po]: why 살던->살았던?

**Comment by youknowone** [comment]
Will you split code PR and localization PR? They require very different reviews and reviewers.

**Comment by gitupofftheflooranddosomework** [comment]
Should be closed as they are not split and can be corrected at a later time when a translator is available

**Pull request closed without merging**